src/components/chip-group/chip-group.e2e.ts

```diff
@@ -0,0 +1,382 @@
+    await page.waitForChanges();
+    expect(eventSpy).toHaveReceivedEventTimes(0);
+    expect(chip1).not.toHaveAttribute("selected");
+    expect(chip2).not.toHaveAttribute("selected");
+    expect(await element.getProperty("selectedItems")).toEqual([]);
+
+    await chip2.click();
+    await page.waitForChanges();
+    expect(eventSpy).toHaveReceivedEventTimes(0);
+    expect(chip1).not.toHaveAttribute("selected");
+    expect(chip2).not.toHaveAttribute("selected");
+    expect(await element.getProperty("selectedItems")).toEqual([]);
+
+    await chip2.click();
+    await page.waitForChanges();
+    expect(eventSpy).toHaveReceivedEventTimes(0);
+    expect(chip1).not.toHaveAttribute("selected");
+    expect(chip2).not.toHaveAttribute("selected");
+    expect(await element.getProperty("selectedItems")).toEqual([]);
+  });
+
+  it("navigation with keyboard works as expected", async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      `<calcite-chip-group label="test-label" selection-mode="single">
+      <calcite-chip id="chip-1"></calcite-chip>
+      <calcite-chip id="chip-2"></calcite-chip>
+      <calcite-chip id="chip-3"></calcite-chip>
+      <calcite-chip id="chip-4"></calcite-chip>
+      <calcite-chip id="chip-5"></calcite-chip>
+      </calcite-chip-group>`
+    );
+
+    const chip1 = await page.find("#chip-1");
+    const chip2 = await page.find("#chip-2");
+    const chip3 = await page.find("#chip-3");
+    const chip4 = await page.find("#chip-4");
+    const chip5 = await page.find("#chip-5");
+
+    await chip1.click();
+    await page.waitForChanges();
+    expect(await page.evaluate(() => document.activeElement.id)).toEqual(chip1.id);
+
+    await page.keyboard.press("ArrowRight");
+    await page.waitForChanges();
+    expect(await page.evaluate(() => document.activeElement.id)).toEqual(chip2.id);
+
+    await page.keyboard.press("ArrowRight");
+    await page.waitForChanges();
+    expect(await page.evaluate(() => document.activeElement.id)).toEqual(chip3.id);
+
+    await page.keyboard.press("End");
+    await page.waitForChanges();
+    expect(await page.evaluate(() => document.activeElement.id)).toEqual(chip5.id);
+
+    await page.keyboard.press("ArrowLeft");
+    await page.waitForChanges();
+    expect(await page.evaluate(() => document.activeElement.id)).toEqual(chip4.id);
+
+    await page.keyboard.press("Home");
+    await page.waitForChanges();
+    expect(await page.evaluate(() => document.activeElement.id)).toEqual(chip1.id);
+
+    await page.keyboard.press("ArrowLeft");
+    await page.waitForChanges();
+    expect(await page.evaluate(() => document.activeElement.id)).toEqual(chip5.id);
+
+    await page.keyboard.press("ArrowRight");
+    await page.waitForChanges();
+    expect(await page.evaluate(() => document.activeElement.id)).toEqual(chip1.id);
+  });
+
+  it("when chips are selectable, and a chip is focused, using tab will focus the close button", async () => {
```

Comment:
Interesting - could be good to look into this. May affect Combobox chip dismissal - I wonder if it could mistakenly be hit as users try to exit Combobox?

Good idea though, will leave this comment and add to a follow-up issue after PR is in.